src/wakatime.ts

```diff
@@ -467,7 +467,7 @@ export class WakaTime {
       if (code == 0) {
         if (output && this.showStatusBar && this.showCodingActivity) {
           this.statusBar.text = `$(clock) ${output}`;
-          this.statusBar.tooltip = `WakaTime: You coded ${output.trim()} today.`;
+          this.statusBar.tooltip = `WakaTime: Today you have spent ${output.trim()}.`;
```

Comment:
```suggestion
          this.statusBar.tooltip = `WakaTime: You have spent ${output.trim()} today.`;
```
Might be better as it matches the original tooltip closer.